docs/howto/how-to-build-compiler.md

```diff
@@ -212,3 +213,35 @@ NOTE: this assumes
 - host and target have same directoy structure
 - should copy `build` folder to target or
 - mounting `ONE` folder with NFS on the target would be simple
+
+## Known issues
+There's a potential known build error when attempting to cross-compile for ARM32 using GCC 10.5.
+You might encounter an error:
+`comparison of unsigned expression in ‘< 0’ is always false [-Werror=type-limits]` reported from
```

Comment:
this is rendered like

![image](https://github.com/user-attachments/assets/5295da37-c6d0-475d-be6a-38200bb1e0ad)

is this your intention?

Reply:
![obraz](https://github.com/user-attachments/assets/b834934f-e012-4b93-977e-a590d9533f45)
This is how it looks for me and it matches how the master branch version is rendered https://github.com/Samsung/ONE/blob/master/docs/howto/how-to-build-compiler.md

What is the issue exactly?

Reply:
> What is the issue exactly?

Source:
> You might encounter an error: 
comparison of unsigned expression in `

Rendered result:

> You might encounter an error: comparison of unsigned expression in

Reply:
The line break after "You might encounter an error:" is just to avoid lines longer than 100 characters in the source code - https://github.com/Samsung/ONE/pull/14535#discussion_r1919875763

It's ok that the error message is rendered in the same line.

Reply:
OK